spec/katalyst/basic/auth_spec.rb

```diff
@@ -4,8 +4,4 @@
   it "has a version number" do
     expect(Katalyst::Basic::Auth::VERSION).not_to be nil
   end
```

Comment:
I'd like to see tests for `enabled?`, `default_username`, `default_password`

Reply:
spec tests added